src/c2goto/CMakeLists.txt

```diff
@@ -59,16 +61,21 @@ function(mangle_clib output)
     else()
       set(inputs_c ${c2goto_library_files})
     endif()
-    set(CMD c2goto ${multiarch_flags} ${OS_C2GOTO_FLAGS}
+
+    set(run_c2goto c2goto ${multiarch_flags} ${OS_C2GOTO_FLAGS}
                    ${inputs_c} ${in_flags} --output ${out_goto})
+
+    # Convert a list of C operational models (e.g. *.c) into a single goto binary file (e.g. clib32.goto) at CMake build time
     add_custom_command(OUTPUT ${out_goto}
-      COMMAND ${CMD}
+      COMMAND ${run_c2goto}
       DEPENDS c2goto ${c2goto_library_files} ${c2goto_libm_files} ${c2goto_header_files}
       WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
       COMMENT "Generating libc model ${out_goto}"
       VERBATIM
       )
 
+    # Convert the content of goto binary file (e.g. clib32.goto) into an unsigned int array at CMake build time
+    # The resulting array will be written in the correpsonding C file (e.g. clib32.c) in the current binary directory.
```

Comment:
"into an unsigned int array" is misleading (since it's actually of type `char`): maybe just write "into a C source file containing the content of the binary".